src/github/credentials.ts

```diff
@@ -3,7 +3,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import * as Octokit from '@octokit/rest';
+import Octokit = require('@octokit/rest');
```

Comment:
For octokit, the module ships with a .d.ts typings file, so I think the normal import syntax is fine

Reply:
If I'm understanding correctly, the issue isn't the lack of a typings file, but rather the export syntax used within it:

```ts
declare namespace Octokit {
  // ...
}


declare class Octokit {
  // ...
}

export = Octokit
```

> The export = syntax specifies a single object that is exported from the module. This can be a class, interface, namespace, function, or enum.
>
> When exporting a module using `export =`, TypeScript-specific `import module = require("module")` must be used to import the module.

From a little investigation, it seems like importing it with `import * as Octokit from '@octokit/rest'` will import only the namespace definitions but not the class. Granted, I doubt we're using the Octokit class, so I don't think it matters much. `assert` is a similar case; if you `import * as assert from 'assert'` then you don't get the `assert(true)` function form.

Is any of this inaccurate? I'm slowly learning my way around TypeScript imports, advice is appreciated 😄